pydis_site/apps/content/README.md

```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
+relatively little code in there, most of it is concerned with serving our
+content.
+
+The markdown files hosting our content can be found in the
+[`resources/`](./resources) directory, and contain
+
+
+## Contributing pages
+
+Contributing pages is covered extensively in our online guide, which you can
+find
+[here](https://www.pythondiscord.com/pages/guides/pydis-guides/how-to-contribute-a-page/)
+or alternatively read it directly at
```

Comment:
Split the sentence up so it doesn't get so long.
```suggestion
Alternatively, read it directly at
```

Reply:
ca2c583b15d46d7821a03ec047678429f96c7e03